aiohttp/web_request.py

```diff
@@ -613,7 +613,10 @@ def body_exists(self) -> bool:
                 field_ct = field.headers.get(hdrs.CONTENT_TYPE)
 
                 if isinstance(field, BodyPartReader):
-                    if field.filename and field_ct:
+                    # Note that according to RFC 7578, the Content-Type header
+                    # is optional, even for files, so we can't assume it's present.
```

Comment:
Plz fit this line in 79 chars: https://github.com/aio-libs/aiohttp/pull/4090/checks?check_run_id=235617239#step:7:7

Reply:
Done, thanks!